dev/devicelab/manifest.yaml

```diff
@@ -76,13 +76,12 @@ tasks:
     stage: devicelab_win
     required_agent_capabilities: ["windows/android"]
 
-  # TODO(jonahwilliams): re-enable with https://github.com/flutter/flutter/issues/39597
-  # codegen_integration_win:
-  #   description: >
-  #     Runs codegeneration and verifies that it can execute
-  #     correctly.
-  #   stage: devicelab_win
-  #   required_agent_capabilities: ["windows/android"]
+  codegen_integration_win:
+    description: >
+      Runs codegeneration and verifies that it can execute
+      correctly.
+    stage: devicelab_win
+    required_agent_capabilities: ["windows/android"]
```

Comment:
nit: might be good to do this in a separate change so that it can be independently reverted more easily.

Reply:
Maybe same for the docker change. I'm not familiar with the original docs issue, but I'm guessing it was a flake?